.papr.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -xeuo pipefail
+
+export GOPATH=$HOME/gopath
+export PATH=$HOME/gopath/bin:$PATH
+export GOSRC=$HOME/gopath/src/github.com/projectatomic/libpod
+
+(mkdir -p $GOSRC && cd /code && cp -r . $GOSRC)
+
+dnf install -y \
```

Comment:
I this ever going to run against a RHEL system?  Hmmm....n/m it's probably not a problem that needs solving now.